components/purchase_modal/purchase.scss

```diff
@@ -0,0 +1,165 @@
+            font-weight: 600;
+        }
+
+        .normal-text {
+            font-weight: normal;
+        }
+
+        .LHS {
+            width: 25%;
+
+            .title {
+                font-size: 24px;
+            }
+
+            .image {
+                padding: 32px 0px;
+            }
+        }
+
+        .central-panel {
+            width: 50%;
+        }
+
+        .RHS {
+            width: 25%;
+            position: sticky;
+
+            .price-container {
+                font-weight: normal;
+                padding: 24px;
+                border: 1px solid rgba(0, 0, 0, 0.08);
+                box-sizing: border-box;
+                box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.08);
+                border-radius: 4px;
+                background-color: var(--center-channel-bg);
+                margin-bottom: 24px;
+                min-width: 270px;
+
+                .footer-text {
+                    font-size: 14px;
+                    margin-bottom: 24px;
+                }
+            }
+
+            .price-text {
+                font-size: 32px;
+                font-weight: 600;
+                padding: 16px 0px
+            }
+
+            .monthly-text {
+                font-size: 14px;
+                font-weight: normal;
+            }
+        }
+
+        .waves {
+            position: absolute;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            top: 0;
+            z-index: -1;
+            // opacity:0.08;
+
+            stop {
+                stop-color: var(--button-bg);
+            }
+        }
+
+        .blue-dots {
+            position: absolute;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            top: 0;
+            z-index: -1;
+
+            // this opacity makes it invisible
+            // opacity: 0.16;
+        }
+
+        .logo {
+            position: absolute;
+            bottom: 0px;
+            left: 0px;
+        }
+
+        button {
+            background: #0058CC;
+            width: 100%;
+            height: 40px;
+            font-weight: 600;
+            font-size: 14px;
+            border-radius: 4px;
+            background: var(--button-bg);
+            color: var(--button-color);
+            border: 0;
+            // margin: 24px 0px;
+
+            &:disabled {
+                background: var(--center-channel-color-08);
+                color: var(--center-channel-color-32);
+            }
+        }
+
+
+        /// ?
+        .SignUpRHS {
+            position: relative;
+            margin-top: 29px;
+            width: 25%;
+            z-index: 2;
+        }
+
+        .SignUpRHS-terms {
+            position: absolute;
+            bottom: 0;
+            opacity: 0.6;
+            font-size: 16px;
+            line-height: 24px;
+            margin-bottom: 42px;
+        }
+
+        .SignUpRHS-link {
+            color: #22406D;
+            margin-left: 32px;
+        }
```

Comment:
I believe we can remove this, this was part of the migration from CWS but not used here